docker/witnet-rust/migrator.sh

```diff
@@ -20,7 +20,7 @@ function migrate {
   log "Ensuring that storage folder '$WITNET_STORAGE_FOLDER' does exist" &&
   mkdir -p "$WITNET_STORAGE_FOLDER" &&
   log "Moving configuration files into configuration folder '$WITNET_CONFIG_FOLDER'" &&
-  mv "$CONFIG_FILE" "$WITNET_CONFIG_FOLDER/witnet.toml" &&
+  mv -n "$CONFIG_FILE" "$WITNET_CONFIG_FOLDER/witnet.toml" &&
```

Comment:
Would it be possible to add a log line indicating that the config file was not overwritten?

Reply:
Sure!